SMelidoni/coinpulse-crypto-app
# hotfix/FAG-index-tooltip-styling

## Description of Changes

Improved the responsiveness of the FAG index tooltip by updating its mobile and tablet styles. This change provides a more user-friendly experience on smaller screens.

## Type of Change


## Describe the problem this PR solves and your solution.

Previously, the FAG index tooltip was not optimised for mobile and tablet screen sizes, potentially leading to display issues. This PR updates the styles of the tooltip to ensure it displays correctly and is easily readable across various screen sizes.

## Testing Steps

1. Open the application on a mobile or tablet device (or use a browser's device emulation).
2. Navigate to the FAG index section.
3. Hover over or 'i' on the tooltip icon.
4. Verify that the tooltip displays correctly and text is legible.
5. Adjust the browser/device width to test different breakpoints (e.g., less than 600px and less than 480px) and ensure the tooltip adapts appropriately.

## Screenshots

_Add any images that will help this PR to be reviewed_

N/A

## Checklist
